piptools/cache.py

```diff
@@ -51,6 +51,8 @@ class DependencyCache(object):
     def __init__(self, cache_dir=None):
         if cache_dir is None:
             cache_dir = CACHE_DIR
+        if not os.path.isdir(cache_dir):
+            os.makedirs(cache_dir)
```

Comment:
Ah, yes, that's why I didn't see it, it's a Python 3 option that isn't in Python 2.